src/main/java/org/jitsi/jicofo/util/JingleOfferFactory.java

```diff
@@ -322,6 +344,13 @@ private void addVideoToContent(ContentPacketExtension content,
         // a=rtcp-fb:100 goog-remb
         vp8.addRtcpFeedbackType(createRtcpFbPacketExtension("goog-remb", null));
 
+        if (ENABLE_TCC)
+        {
+            // a=rtcp-fb:100 transport-cc
+            vp8.addRtcpFeedbackType(
```

Comment:
i know we have these all organized as per-codec, but for things like this i wonder if it would be best to have a single `if (ENABLE_TCC)` block which adds it for all codecs (perhaps the codecs should even be added to an array or something so it could just iterate over it rather than having to hard-code each one which was created above?)

Reply:
Will do, make more sense.